feature-libs/user/account/components/myaccount-view-name/myaccount-view-name.component.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component } from '@angular/core';
+import { LoginComponent } from '../login';
+
+@Component({
+  selector: 'cx-myaccount-view-name',
+  templateUrl: './myaccount-view-name.component.html',
+})
+export class MyaccountViewNameComponent extends LoginComponent {}
```

Comment:
[MEDIUM]
IMHO this component name is unclear. It sounds like the my account view would have it's name. 
Instead I suggest highlighting it's an "user name". 

---

Side note: my account are 2 words, so lets distinguish them by using camelCase/PascalCase for it when needed, e.g. `myAccount` / `MyAccount`.

Reply:
will change it to MyAccountViewUserName . is this ok

Reply:
Sounds good. 

Btw. could you help me understand why in general in this PR we use phrase `MyAccountView` instead of `MyAccount`? Perhaps there were reasons to do it. What pros/cons of each approach do you see?

Reply:
MyAccountView was suggested by our team UA and hence we maintained the same